.gitignore

```diff
@@ -0,0 +1,16 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```

Comment:
Tip: put the entire idea folder in gitignore. You will rarely need to sync anything here with somebody else